HarryPotter/Screens/House/ViewController/HouseViewController.swift

```diff
@@ -78,6 +96,14 @@ extension HouseViewController {
         let currentCharacter = viewModel.characters[indexPath.row]
         cell.myLabel.text = currentCharacter.name
         cell.backgroundColor = viewModel.backgroundColor
+        cell.selectionStyle = .none
+        //cell.houseViewController = self
```

Comment:
Yep, so that's right, you can delete the cell.houseViewController, and saving is done using closure, below here